src/data/contacts.ts

```diff
@@ -419,11 +419,6 @@ export const contacts: Contacts = {
     email: 'SteveBowman@utah.gov',
     phone: '801-537-3304',
   },
-  print_proxy: {
-    name: 'Scott Davis',
-    agency: 'UGRC',
-    email: 'stdavis@utah.gov',
-  },
```

Comment:
what was the reasoning to pull out of here?

Reply:
Because I had everything going to ugrc-developers@utah.gov for print proxy.

Reply:
Are you ok with using the contact component and updating the contacts to point to the generic ugrc developers? We have a "rule" not to have mailto links in pages. 

I think we should consolidate our work emails for the group in other places. I can put together that pull request if you're happy with that.

Reply:
I didn't realize that the mailto rule applied to generic accounts but I'm good with that. I'll submit a quick PR to fix this page.

Reply:
thanks. https://github.com/agrc/gis.utah.gov#contact-information